src/month_calendar/month_view.es

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { arrayOf } from 'nebenan-helpers/lib/data';
+import clsx from 'clsx';
+import { formatNumber as pad } from 'nebenan-helpers/lib/formatters';
```

Comment:
i don't think yuo really need nebenan-helpers dependency here, both of these functions are either built-in or simple to write (considering IE11 drop)

Array.form(new Array(size)) and string.padStart(charCount, filler)